### PR TITLE
Fix CheckStyle issues in util/OpenCompute.java

### PR DESCRIPTION
**Pull request by Pr0methean**

### CODE REVIEW

File: src/main/java/net/glowstone/util/OpenCompute.java [hunk 1]
```diff
@@ -92,32 +122,21 @@ public static void initContext(CLPlatform platform) {
...
-        int r = globalSize % localWorkSize;
-        if (r != 0) {
-            globalSize += localWorkSize - r;
-        }
-        return globalSize;
+        return getGlobalSize(size, getLocalSize());
     }
 
+    /**
+     * Calculates the number of work groups.
```
Comment (mastercoms) [inline src/main/java/net/glowstone/util/OpenCompute.java:135]: New line between description and params.

File: src/main/java/net/glowstone/util/OpenCompute.java [hunk 2]
```diff
@@ -36,15 +49,18 @@ public static CLProgram getProgram(String name) {
                         programs.put(name, program);
                         return program;
                     } catch (IOException ex) {
-                        GlowServer.logger.log(Level.WARNING, "Could not load custom OpenCL program. Trying builtins.", ex);
+                        GlowServer.logger.log(Level.WARNING,
+                                "Could not load custom OpenCL program. Trying builtins.", ex);
```
Comment (mastercoms) [inline src/main/java/net/glowstone/util/OpenCompute.java]: Could you remove the "Trying builtins" part, as that seems to be false.

File: src/main/java/net/glowstone/util/OpenCompute.java (hunk 1 above)
Comment (Pr0methean) [reply src/main/java/net/glowstone/util/OpenCompute.java:135]: Done.

File: src/main/java/net/glowstone/util/OpenCompute.java (hunk 2 above)
Comment (Pr0methean) [reply src/main/java/net/glowstone/util/OpenCompute.java]: Done.

File: src/main/java/net/glowstone/util/OpenCompute.java [hunk 3]
```diff
@@ -135,6 +155,10 @@ public static int getLocalSize(int max) {
         return Math.min(device.getMaxWorkGroupSize(), max);
```
Comment (mastercoms) [inline src/main/java/net/glowstone/util/OpenCompute.java:166]: Are these getLocalSize methods not supposed to have javadocs in this PR?

File: src/main/java/net/glowstone/util/OpenCompute.java [hunk 4]
```diff
@@ -12,18 +12,31 @@
...
     private static CLCommandQueue queue;
     private static HashMap<String, CLProgram> programs;
     private static HashMap<CLProgram, HashMap<String, CLKernel>> kernels;
 
+    /**
+     * Returns an OpenCL program, loading it synchronously if it's not in cache.
+     *
+     * @param name the program filename
+     * @return the OpenCL program, or null if there isn't a valid program with that name
+     *
```
Comment (aramperes) [inline src/main/java/net/glowstone/util/OpenCompute.java]: Remove the blank line here

File: src/main/java/net/glowstone/util/OpenCompute.java (hunk 3 above)
Comment (Pr0methean) [reply src/main/java/net/glowstone/util/OpenCompute.java:166]: Added, although CheckStyle didn't demand them.

File: src/main/java/net/glowstone/util/OpenCompute.java (hunk 4 above)
Comment (Pr0methean) [reply src/main/java/net/glowstone/util/OpenCompute.java]: Done.

File: src/main/java/net/glowstone/util/OpenCompute.java (hunk 1 above)
Comment (aramperes) [reply src/main/java/net/glowstone/util/OpenCompute.java:135]: @Pr0methean please do the same thing with other Javadocs in this PR.

Comment (Pr0methean) [reply src/main/java/net/glowstone/util/OpenCompute.java:135]: Done in 283d634

File: src/main/java/net/glowstone/util/OpenCompute.java [hunk 5]
```diff
@@ -92,32 +121,22 @@ public static void initContext(CLPlatform platform) {
...
-    public static CLDevice getDevice() {
-        return device;
-    }
-
-    public static CLCommandQueue getQueue() {
-        return queue;
-    }
-
+    /**
+     * Calculates the number of work groups.
```
Comment (momothereal) [inline src/main/java/net/glowstone/util/OpenCompute.java]: Here too

**Review by aramperes — APPROVED** [review]

**Pull request merged**